### PR TITLE
Fix #170: vagrant-service-manager version 1.0.1 release

### LINKED ISSUE #170: vagrant-service-manager 1.0.1 release
- [x] #160 
- [x] #166 
- [x] #170
- [x] https://github.com/projectatomic/vagrant-service-manager/pull/164

would like to be part of the release:
- [ ] - https://github.com/projectatomic/vagrant-service-manager/issues/173

### PR DESCRIPTION
**Pull request by navidshaikh**
Fixes #170

### CODE REVIEW

**Comment by praveenkumar** [comment]
LGTM.

File: vagrant-service-manager.spec [hunk 1]
```diff
@@ -88,6 +88,13 @@ popd
 %{vagrant_plugin_instdir}/.gitattributes
 
 %changelog
+* Thu Apr 12 2016 Navid Shaikh - 1.0.1-1
```
Comment (brgnepal) [inline vagrant-service-manager.spec]: `Tue` I believe :smile:

**Comment by brgnepal** [comment]
LGTM except my comment on Day.

**Comment by LalatenduMohanty** [comment]
LGTM

**Pull request merged**